src/main/java/org/folio/service/orders/utils/StatusUtils.java

```diff
@@ -28,16 +28,22 @@ public static boolean areAllPoLinesCanceled(List<PoLine> poLines) {
     return poLines.stream().allMatch(StatusUtils::isStatusCanceledPoLine);
   }
 
-  public static boolean changeOrderStatus(PurchaseOrder purchaseOrder, List<PoLine> poLines) {
+  public static boolean changeOrderStatusForOrderUpdate(PurchaseOrder purchaseOrder, List<PoLine> poLines) {
     if (toBeCancelled(purchaseOrder, poLines)) {
-      purchaseOrder.setWorkflowStatus(PurchaseOrder.WorkflowStatus.CLOSED);
-      purchaseOrder.setCloseReason(new CloseReason().withReason(REASON_CANCELLED));
-      return true;
+      return closeOrder(purchaseOrder, REASON_CANCELLED);
     }
     if (toBeClosed(purchaseOrder, poLines)) {
-      purchaseOrder.setWorkflowStatus(PurchaseOrder.WorkflowStatus.CLOSED);
-      purchaseOrder.setCloseReason(new CloseReason().withReason(REASON_COMPLETE));
-      return true;
+      return closeOrder(purchaseOrder, REASON_COMPLETE);
+    }
+    return false;
+  }
+
+  public static boolean changeOrderStatusForPoLineUpdate(PurchaseOrder purchaseOrder, List<PoLine> poLines) {
+    if (toBeCancelled(purchaseOrder, poLines)) {
+      return closeOrder(purchaseOrder, REASON_CANCELLED);
+    }
+    if (toBeClosed(purchaseOrder, poLines)) {
+      return closeOrder(purchaseOrder, REASON_COMPLETE);
     }
     if (toBeReopened(purchaseOrder, poLines)) {
```

Comment:
we need to remain toBeReopened() from PO Line update, in this case for example when some PO Line paymentStatus changed from FULLY_PAID to AWAITING_PAYMENT we should reopen the order. also related ticket: https://folio-org.atlassian.net/browse/MODORDERS-1143